docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -35,9 +35,9 @@ The icon shouldn't fill too much or too little of the canvas. Avoid filling the
 
 The icon should have good contrast on dark and light backgrounds. In particular, avoid single-color glyphs and the like since they can be very hard to see if the background color is similar. Dark colors near the edges are also a problem, since shadows don't help much with contrast on dark backgrounds.
 
-### Not too much detail
+### Not too much or too little detail
```

Comment:
Btw yikes, this will change the url, which I need to change on the website too :/